providers/azure/resources/azure.lr

```diff
@@ -1063,6 +1063,22 @@ private azure.subscription.webService.appsite @defaults("id name location") {
   diagnosticSettings() []azure.subscription.monitorService.diagnosticsetting
   // List of functions for the web app site
   functions() []azure.subscription.webService.function
+  // FTP publishing method policies
+  ftp() azure.subscription.webService.appsite.basicPublishingCredentialsPolicies
+  // SCM publishing method policies
+  scm() azure.subscription.webService.appsite.basicPublishingCredentialsPolicies
+}
+
+// Azure AppSite basic publishing credentials policies
+private azure.subscription.webService.appsite.basicPublishingCredentialsPolicies @defaults("id allow") {
+  // Resource ID
+  id string
+  // Resource name
+  name string
+  // Resource type
+  type string
+  // Publishing method policy
```

Comment:
I see that this is a bool but from the description it isn't clear to me what it's setting. Is it just whether to allow publishing?

Reply:
@misterpantz this is the data we get https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/resourcemanager/appservice/armappservice/models.go#L1870-L1892

I wonder if you have a suggestion I can use 😄

Reply:
Super! I recommend:
```suggestion
  // Whether to allow access to a publishing method
```